dist/src/main/java/io/camunda/webapps/WebappsModuleConfiguration.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/** Configuration for shared spring components used by webapps */
+@Configuration(proxyBeanMethods = false)
+@ComponentScan(basePackages = "io.camunda.webapps")
+@EnableAutoConfiguration
+public class WebappsModuleConfiguration {}
```

Comment:
Let's depend this on the profiles `tasklist` or `operate` as done with the app-specific configuration classes.

Reply:
Done! `WebappsModuleConfiguration` is added in the sources used in StandaloneOperate, StandaloneTasklist and StandaloneCamunda. It is also enabled with `camunda.webapps.enabled` property set by `WebappsConfigurationInitializer`